packages/web/src/scss/components/Dropdown/_theme.scss

```
@@ -1,5 +1,6 @@
 @use '@tokens' as tokens;
 
+$breakpoints: tokens.$breakpoints;
 $padding: tokens.$space-600;
```

We will need this for the dropdown offset as noted above:

```suggestion
$offset: tokens.$space-400;
$padding: tokens.$space-600;
```